src/odemis/driver/hamamatsurx.py

```diff
@@ -579,7 +580,8 @@ def __init__(self, name, role, parent, daemon=None, **kwargs):
         self.parent = parent
         self.location = "Streakcamera"  # don't change, internally needed by HPDTA/RemoteEx
 
-        self._hwVersion = parent.DevParamGet(self.location, "DeviceName")
+        # needs to be a string for tiff export!
```

Comment:
Actually, it needs to be a string to follow the rest of Odemis expectation that this metadata/property is a string.
Python doesn't mind about it, but the rest of Odemis does.